.typos.toml

```diff
@@ -0,0 +1,26 @@
+[default.extend-words]
+k8s = "k8s"
+KServe = "KServe"
+vLLM = "vLLM"
+IST = "IST"
+parm = "parm"
+Ded = "Ded"
```

Comment:
Q: is this due to ./docs/architecture.md:`of the keys are checked and the results are ANDed together.`?
If so, I wonder if that's the right addition.